continuous_integration/kubernetes/install.sh

```diff
@@ -25,5 +25,9 @@ if [[ "$TRAVIS" != "true" ]]; then
     eval $(minikube docker-env)
 fi
 pushd $git_root
+
+echo "Installing chartpress"
+pip install --default-timeout=100 git+https://github.com/jupyterhub/chartpress.git
```

Comment:
This can just be `chartpress` now as it is on PyPI.

```suggestion
pip install --default-timeout=100 chartpress
```